### PR TITLE
Update 5.0.4

### PR DESCRIPTION
**Pull request by gcaguilar**

### CODE REVIEW

File: com.axosoft.GitKraken.appdata.xml [hunk 1]
```diff
@@ -78,7 +78,8 @@
     </categories>
 
     <releases>
-        <release date="2019-03-27" version="5.0.3"/>
+        <release date="2019-03-27" version="5.0.4"/>
```
Comment (Lctrs) [inline com.axosoft.GitKraken.appdata.xml]: Please use today's date. I chose to put the date when I update the flatpak wrapper, and not when upstream release a new version.

File: com.axosoft.GitKraken.appdata.xml [hunk 2]
```diff
@@ -78,7 +78,8 @@
...
+        <release date="2019-03-26" version="5.0.3"/>
```
Comment (Lctrs) [inline com.axosoft.GitKraken.appdata.xml]: Also revert this one.

**Comment by gcaguilar** [comment]
@Lctrs Fixed

**Comment by Lctrs** [comment]
Thank you @gcaguilar !

**Pull request merged**